MapboxNavigation/PollyVoiceController.swift

```diff
@@ -42,7 +42,8 @@ public class PollyVoiceController: RouteVoiceController {
         
         let routeProgresss = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationRouteProgressKey] as! RouteProgress
         let userDistances = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationDistanceToEndOfManeuverKey] as! CLLocationDistance
-        let instruction = spokenInstructionFormatter.string(routeProgress: routeProgresss, userDistance: userDistances, markUpWithSSML: true)
+        let voiceInstructionIndex = routeProgresss.currentLegProgress.voiceInstructionIndex
+        let instruction = routeProgresss.currentLegProgress.currentStep.instructionsSpokenAlongStep?[voiceInstructionIndex].ssml ?? spokenInstructionFormatter.string(routeProgress: routeProgresss, userDistance: userDistances, markUpWithSSML: true)
```

Comment:
Is `userDistance` still needed?

Reply:
I'd like to keep `spokenInstructionFormatter.string` around for testing and comparison, so yes still needed.